etest_test/unit_test/overlay_test.py

```diff
@@ -8,36 +8,31 @@
 import logging
 import os
 import tempfile
-from typing import Set
+import unittest.mock
 
-from etest import overlay
-from etest_test.common_test import BaseEtestTest
+import etest.overlay as sut
 from etest_test.fixtures_test import FIXTURES_DIRECTORY
 
 logger = logging.getLogger(__name__)
 
 
-class BaseOverlayTest(BaseEtestTest):
+@unittest.mock.patch.object(sut, "ebuild")
+class BaseOverlayTest(unittest.TestCase):
     """Overlay test."""
 
-    mocks_mask: Set = set()
-    mocks: Set = set()
-
     def setUp(self):
```

Comment:
Is this supposed to be camelCase? Im guessing its an inherited method.

Reply:
It is, it's inherited from unittest.TestCase which follows the jUnit conventions including camelCase.